fuel-vm/src/tests/memory.rs

```diff
@@ -102,6 +103,125 @@ fn test_lb() {
     assert_eq!(1, result);
 }
 
+#[test_case(op::sb, 0, 0, 0 => RunResult::Success([0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]))]
```

Comment:
Please consider grouping some of these test cases by labels or comments to make it easier to understand the intention and coverage of these tests.

Reply:
Done in b65d73ae9